oracle/README.md

```diff
@@ -52,37 +52,14 @@ When you run `datadog-agent info` you should see something like the following:
 
 ### Compatibility
 
-The oracle check is currently compatible with the linux and darwin-based OS
+The Oracle check is currently compatible with Linux and macOS.
 
 ## Data Collected
 ### Metrics
 See [metadata.csv](https://github.com/DataDog/integrations-core/blob/master/oracle/metadata.csv) for a list of metrics provided by this integration.
 
 ### Events
-The oracle check does not include any event at this time.
+The Oracle Database check does not include any event at this time.
 
 ### Service Checks
-The oracle check does not include any service check at this time.
-
-## Troubleshooting
-
-If you have any questions about Datadog or a use case our [Docs](https://docs.datadoghq.com/) didn’t mention, we’d love to help! Here’s how you can reach out to us:
-
-### Visit the Knowledge Base
-
-Learn more about what you can do in Datadog on the [Support Knowledge Base](https://datadog.zendesk.com/agent/).
-
-### Web Support
-
-Messages in the [event stream](https://app.datadoghq.com/event/stream) containing **@support-datadog** will reach our Support Team. This is a convenient channel for referencing graph snapshots or a particular event. In addition, we have a livechat service available during the day (EST) from any page within the app.
-
-### By Email
-
-You can also contact our Support Team via email at [support@datadoghq.com](mailto:support@datadoghq.com).
-
-### Over Slack
-
-Reach out to our team and other Datadog users on [Slack](http://chat.datadoghq.com/).
-
-## Further Reading
-Learn more about infrastructure monitoring and all our integrations on [our blog](https://www.datadoghq.com/blog/)
\ No newline at end of file
+The Oracle Database check does not include any service check at this time.
```

Comment:
This doesn't look to be correct. See: https://github.com/DataDog/integrations-core/blob/master/oracle/check.py#L90

CC: @marcweisman

Reply:
Also incorrect - we do submit a service check.